docs(gatsby-source-contentful): Add info to readme on use with new image plugin (#29297)
gatsbyjs/gatsby: packages/gatsby-source-contentful/README.md

When querying images you can use the `fixed`, `fluid` or `resize` nodes to get different sizes for the image (for example for using [`gatsby-image`](https://www.gatsbyjs.org/packages/gatsby-image/)). Their usage is documented at the [`gatsby-plugin-sharp`](https://www.gatsbyjs.org/packages/gatsby-plugin-sharp/) package. The only difference is that `gatsby-source-contentful` also allows setting only the `width` parameter for these node types, the height will then automatically be calculated according to the aspect ratio.

## Using the new Gatsby image plugin

You can now use the beta gatsby-plugin-image to display high-performance, responsive images from Contentful. This is the replacement for gatsby-image, and is currently in beta, but can help deliver improved performance and a cleaner API. Support in gatsby-source-contentful is still experimental.

> To use it, you must install the plugins:

Reviewer: ```suggestion
To use it:

 1.  Install the plugins:
```

```shell
npm install gatsby-plugin-image gatsby-plugin-sharp
```

> ...and then add them to your `gatsby-config.js`:

Reviewer: ```suggestion
2. Add the plugins to your `gatsby-config.js`:
```

```javascript
module.exports = {
  plugins: [
    `gatsby-plugin-sharp`,
    `gatsby-plugin-image`,
    // ...etc
  ],
}
```

> You can then query for image data using the new `gatsbyImageData` resolver:

Reviewer: ```suggestion
3. You can then query for image data using the new `gatsbyImageData` resolver:
```
